core/src/main/resources/hudson/model/StringParameterDefinition/config.jelly

```diff
@@ -35,4 +35,7 @@ THE SOFTWARE.
     <f:entry title="${%Description}" help="/help/parameter/description.html">
         <f:textarea name="parameter.description" value="${instance.description}" codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="/markupFormatter/previewDescription" />
     </f:entry>
-</j:jelly>
\ No newline at end of file
+    <f:entry field="trim">
+        <f:checkbox default="false" title="${%Trim the string}"/>
```

Comment:
I'm not sure how universal this term is; probably deserving of a (short) help file, explaining what exactly gets removed.